### ISSUE TITLE
Add DWN endpoints to Agent DID and ensure they sync

### ISSUE THREAD

**Issue by csuwildcat** [TBD54566975/web5-js#652]

**Comment by frankhinek** [comment]
@csuwildcat The Agent DID's private key can't be stored in the DWN because otherwise it has no way to get to it to be able to sign messages with the key (which is necessary to read from the DWN).  Additionally, because the Agent's DID is deterministically created from the 12 words, you will have to consider whether this is still a desirable characteristic, and if so how to handle the non-deterministic `service` object that the issue proposes adding.

**Comment by csuwildcat** [comment]
Yeah, meant everything but the private key, which you'd presumably move via 12 word entry in the new wallet
